examples/hotelling_law/app.py

```diff
@@ -108,7 +108,8 @@ def agent_portrayal(agent):
     return portrayal
 
 
-def space_drawer(model, agent_portrayal):
+@solara.component
+def SpaceDrawer(model):
```

Comment:
This function is capitalized like a Class. For my own knowlegde, does `@solara.component` make it Class-like?

Reply:
Aren't they all just objects? ;)

As far as I understood this is mostly a convention. It's best to imagine them as neither a class nor a function but as a solara component.

Reply:
Just to clarify, since I have looked it up by now. @solara.component does indeed return a regular ´class` object (with a a defined __call__ function, so you can call it like a function). Usually they should still be written in snake_case for standard Python formatting, but thats where the solara convention takes over